Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM mono:6.12.0
+
+# Install .NET SDK
+ENV DOTNET_VERSION=5.0
+
+RUN curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin -Channel $DOTNET_VERSION -InstallDir /usr/share/dotnet \
```

Comment:
`sh` and `bash` have a `-s` option specifically for reading from stdin, so you could also do:
```sh
... | bash -s -- -Channel $DOTNET_VERSION ...
```